xml/System.Collections.Frozen/FrozenDictionary`2.xml

```diff
@@ -1111,10 +1133,11 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">The key of the value to get.</param>
+        <param name="value">When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of the value parameter.</param>
```

Comment:
```suggestion
        <param name="value">When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of <paramref name="value" />.</param>
```